playwright/features/pub_yearFilter.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Publication Year filter test
+
+    Scenario: As a user I should be able to filter searches by publication year and see relevent results
+        Given I want to filter by publication year
```

Comment:
I'm thinking we should change these statements slightly so they make a little more sense to someone reading the tests for the first time, like `Given I am on the home page` and then `When I search for New York` and then `Then I filter by publication years 1900-1900`

Reply:
good suggestion.  done.